react/src/js/components/Consonant/Infobit/Type/Link.jsx

```diff
@@ -31,12 +33,14 @@ const Link = ({
     href,
     linkHint,
     text,
+    title,
 }) => {
     /**
      **** Authored Configs ****
      */
     const getConfig = useConfig();
     const ctaAction = getConfig('collection', 'ctaAction');
+    const ariaLabel = `${text} ${title}`;
```

Comment:
If `title` is empty (which defaults to ``), you end up with a trailing space in the aria-label. 

Recommend fixing to:
```js
const ariaLabel = title ? `${text} ${title}` : text`
```